src/components/Counter/docs/CounterExtendedExample.tsx

```diff
@@ -52,6 +52,17 @@ export class CounterExtendedExample extends React.Component<{}, State> {
         {...this.props}
         className={classes.root}
       />
+      <h3>xSmall Counter</h3>
+      <Counter
+        inputAriaLabel={'amount'}
+        incrementAriaLabel={'increment'}
+        decrementAriaLabel={'decrement'}
+        onChange={this.handleChange}
+        value={this.state.value}
+        size={CounterSize.xSmall}
+        {...this.props}
+        className={classes.root}
+      />
```

Comment:
I'm not sure why, but this isn't rendering as an xSmall Counter.
It might be because of the `classes.root` class.
this is how it looks:
![image](https://user-images.githubusercontent.com/3277080/103411110-0448bf80-4b77-11eb-855b-cb33eb70684a.png)

Reply:
Was definitely caused because of the 'root' class name. was able to solve by adding '!important' to the overrided properties at xSmall. Is that the right thing to do though?

And one more issue - when the counter value is greater than 9 the input number is cut at the edges due to lack of space. The medium sized counter behaves the same (the number is cut when there is not enough space) but for much bigger numbers. What is the right thing to do in such cases ? Is that a questions for the designers?

Reply:
So no, actually we shouldn't use `!important` in the component.
I would just pass a different className to the xSmall example.
Cause the problem is with the example, and not the code in this case